process faults {5: ({5, 6, 4}, 182469291)}
network faults {6: [{0, 2, 3, 4}, {6, 1}, {5}]}
{6: ("2C2C31C70A31C7249CAA657E352E7348C720E3C63FA445BBD46CBE0959A0C789", {3, 0, 2, 1, 4, 6, 5}), 5: ("FBF4A17FB85F162D1A5C3E1CA1929076C4B604CAED9499456016C3389DC40077", {5, 6, 1, 4, 0, 2, 3}), 7: ("19F97AA588D23EA6227A23FDE5EBF3E929864F2BD4C101954B2444CA4EC3F42C", {1, 4, 6, 3, 2, 0, 5}), 9: ("A963C264C62BADEE4D429A205732806CBD25F1900DFB650FF37C764B47C65F6B", {3, 0, 1, 5, 4, 2, 6}), 8: ("F10647D228EC5A68EC625D153DB4B56F922AB9EE9A6DE9E1B7F4082C81CA24D8", {5, 3, 2, 0, 4, 6, 1}), 10: ("EEF9DA1746A7EA864C2CE0019DA149412DFCEBF39427DECFE48074D990BC44AB", {4, 2, 6, 3, 1, 5, 0})}
7
reason: all committed
done!
